xml/System.Text.RegularExpressions/CaptureCollection.xml

```diff
@@ -670,9 +693,22 @@
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <summary>Returns an enumerator that iterates through the collection.</summary>
+        <returns>An enumerator that can be used to iterate through the collection.</returns>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+The returned <xref:System.Collections.Generic.IEnumerator%601> provides the ability to iterate through the collection by exposing a <xref:System.Collections.Generic.IEnumerator%601.Current%2A> property .You can use enumerators to read the data in a collection, but not to modify the collection.   
+ Initially, the enumerator is positioned before the first element in the collection. At this position, <xref:System.Collections.Generic.IEnumerator%601.Current%2A> is undefined. Therefore, you must call the <xref:System.Collections.IEnumerator.MoveNext%2A> method to advance the enumerator to the first element of the collection before reading the value of <xref:System.Collections.Generic.IEnumerator%601.Current%2A>.   
```

Comment:
Any idea what the weird %601 chars are? Is that some Unicode encoding?

Reply:
It's what you use for the ` character.

It means the type requires a type parameter.

- Notice the API full name ends with `<T>` (with html entities for "<" and ">"):
[dotnet-api-docs/blob/master/xml/System.Collections.Generic/IEnumerator%601.xml#L1](https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Collections.Generic/IEnumerator%601.xml#L1)

- But the DocID changes that to a "`1":
[dotnet-api-docs/blob/master/xml/System.Collections.Generic/IEnumerator%601.xml#L4](https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System.Collections.Generic/IEnumerator%601.xml#L4)

The xml filename uses the "`1", but the URL escapes it to %601.



 has a "-1" [in MS Docs](https://docs.microsoft.com/en-us/dotnet/api/system.collections.generic.ienumerator-1?view=netframework-4.8) at the end, and a back-tick or %601 at the end